scripts/delorean/process-image-manifests

```diff
@@ -197,6 +208,7 @@ process_image() {
    get_file_name_for_new_csv
    update_new_csv
    update_package_yaml
+   update_image_pull_secret
```

Comment:
Hmm, i wonder if this should only happen if the image mirror mapping file has anything in it. We only need to do this if we intend to pull images from the delorean quay.io registry right?

Reply:
Ah true, we only need if we have images to mirror so we should check that. I'll make that change

Reply:
So to do this I thought to just move the update logic to the csv script, as the image processing script runs before the csv script, so it's possible that the image_mirror_mapping might not exist before it runs.

But that relies on a bunch of variables that only exist in the image-manifest script. So i'll need to think about this a bit,.

Reply:
The get_current_csv_file function gets me what I need.